app/lib/features/chat/widgets/bubble_builder.dart

```diff
@@ -151,22 +180,20 @@ class _ChatBubble extends ConsumerWidget {
                                       ActerAvatar(
                                         uniqueId:
                                             message.repliedMessage!.author.id,
+                                        displayName: replyProfile != null
```

Comment:
and considering that `displayName` is used twice here and below, I'd also pull out this assignment above, below the `replyProfile` and reuse the calculated value here and below.